tasks/build.py

```diff
@@ -102,9 +102,28 @@ def get_build_env_cfg(cfg):
     log(f"{fn}(): submit_command '{submit_command}'")
     config_data[config.BUILDENV_SETTING_SUBMIT_COMMAND] = submit_command
 
+    job_handover_protocol = buildenv.get(config.BUILDENV_SETTING_JOB_HANDOVER_PROTOCOL)
     slurm_params = buildenv.get(config.BUILDENV_SETTING_SLURM_PARAMS)
-    # always submit jobs with hold set, so job manager can release them
-    slurm_params += ' --hold'
+    if job_handover_protocol == config.JOB_HANDOVER_PROTOCOL_HOLD_RELEASE:
+        # always submit jobs with hold set, so job manager can release them
+        slurm_params += ' --hold'
+    elif job_handover_protocol == config.JOB_HANDOVER_PROTOCOL_DELAYED_BEGIN:
+        # alternative method to submit without '--hold' and
+        # '--begin=now+5*poll_interval' instead
+        # 1. remove '--hold' if any
+        # 2. add '--begin=now+5*poll_interval'
+        slurm_params = slurm_params.replace('--hold', '')
+        job_manger_cfg = cfg[config.SECTION_JOB_MANAGER]
+        poll_interval = int(job_manger_cfg.get(config.JOB_MANAGER_SETTING_POLL_INTERVAL))
+        slurm_params += f' --begin=now+{5 * poll_interval}'
```

Comment:
I think 5 times the poll interval is actually quite long. In theory, if the job_manager's iteration takes negligible amount of time, 1 * poll_interval would be enough. Of course, we need some safety margin in case the iteration _does_ take longer. But if your job_manager iteration starts to take as long as the poll interval, you've configured too short a poll_interval, no?

I'd say, 2 * poll_interval should be plenty (it's what I initially did in my tests and worked well so far). You _might_ say the maximum of (2* poll_interval, poll_interval + 10), to make sure that it's never shorter than 10 seconds, even for a very short poll interval.

Either that, or make it configurable.